addons/website/views/snippets/snippets.xml

```diff
@@ -142,6 +142,18 @@
               data-unit="px" data-extra-class="rounded"/>
 </template>
 
+<template id="website.snippet_options_shadow">
+    <we-checkbox string="Shadow" data-set-shadow="true" data-no-preview="true" data-name="shadow_active_opt"/>
```

Comment:
Why does this need to be a custom option method? Can't you leave the select-class="shadow" as before so that the shadow values are the ones of the theme by default?

Reply:
It was to remove the shadow when the checkbox was unchecked, is there a clean way to remove the box-shadow with the select-class?

Reply:
You can add another function on top of select-class or override select-class for your specific option.